examples/workflow-glsp/src/di.config.ts

```diff
@@ -89,5 +90,8 @@ export function initializeWorkflowDiagramContainer(
         baseDiv: widgetId,
         hiddenDiv: widgetId + '_hidden'
     });
+
+    container.load(svgMetadataModule);
```

Comment:
I think this module is generic and useful enough to be added to the DEFAULT_MODULES.